app/managers/DialcodeManager.java

```diff
@@ -740,4 +742,23 @@ private boolean verifySchemaAndContextPaths(String strURI) throws IOException {
         return huc.getResponseCode() == HttpURLConnection.HTTP_OK;
     }
 
+    public Response validateContextVocabulary(String contextFile) throws IOException {
+        HashMap contextFileMap = new ObjectMapper().readValue(new URL(contextFile), HashMap.class);
+        HashMap contextMap = (HashMap) contextFileMap.get("@context");
+
+        if(!AppConfig.config.hasPath("jsonld.sb_schema"))
```

Comment:
Update the configuration with a comment this is the mandatory property. Thinks how can we make sure mandatory configuration has define while service start itself.

Reply:
will do that in the devops config file